views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,17 @@
+{{#each beersArray}}
+
+<div class="beerscontainer">
+<div class="beers-content">
+    <div>
+        <img id="beerspage-img" src="{{image_url}}" alt="poster for {{name}}">
```

Comment:
Some beers do not have images. In that case the user can see the little image icon and what you added in "alt". I would suggest to wrap this image tag in an if statement. That way, you can't see the image if there isn't any.